wire-ios/Wire-iOS Tests/ConversationTitleViewTests.swift

```diff
@@ -20,14 +20,14 @@ import XCTest
 @testable import Wire
 import SnapshotTesting
 
-final class ConversationTitleViewTests: XCTestCase {
+final class ConversationTitleViewTests: BaseSnapshotTestCase {
 
     var sut: ConversationTitleView!
-    var conversation: SwiftMockConversation!
```

Comment:
Why do we need to change the type here?

Reply:
I extended `GroupDetailsConversation` protocol with a new `mlsVerificationStatus` property. `SwiftMockConversation` doesn't conform `GroupDetailsConversation` protocol.